tracer/src/Datadog.Trace/Debugger/Expressions/ProbeProcessor.cs

```diff
@@ -448,6 +451,49 @@ private void SetSpanDecoration(DebuggerSnapshotCreator snapshotCreator, ref bool
             }
         }
 
+        private bool TryGetScope([NotNullWhen(true)] out Scope? scope)
```

Comment:
Isn't it a bit weird that scope could be null with a function in the pattern of `TryGetX`?